python_modules/libraries/dagster-looker/dagster_looker_tests/test_asset_specs.py

```diff
@@ -11,18 +11,15 @@
 
 def test_build_looker_asset_specs_as_external_assets() -> None:
     looker_specs = build_looker_asset_specs(project_dir=test_retail_demo_path)
-    my_looker_assets = external_assets_from_specs(looker_specs)
 
-    assert Definitions(assets=[*my_looker_assets])
+    Definitions.validate_loadable(Definitions(assets=looker_specs))
```

Comment:
Out of the scope of this PR, but:

- Does this need to be a static method?
- Feels verbose

```suggestion
    Definitions(assets=looker_specs).validate()
```

Reply:
That was actually my how it worked in my initial version, but switched to a static method based on Nick's feedback: 
https://github.com/dagster-io/dagster/pull/21746#discussion_r1607231433.